catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java

```diff
@@ -437,36 +438,43 @@ private static boolean isAttributeMultiValued(String attribute, MetacardImpl met
         return multiValued;
     }
 
-    private static String getPhoneNumberString(TelephoneNumberType digits) {
-        String phoneNumber;
-        if (StringUtils.isBlank(digits.getExtension())) {
-            phoneNumber = String.format("(%s) %s", digits.getAreaCode(), digits.getNumber());
-        } else {
-            phoneNumber = String.format("(%s) %s extension %s",
-                    digits.getAreaCode(),
-                    digits.getNumber(),
-                    digits.getExtension());
-        }
-
-        return phoneNumber;
+    private static String getPhoneNumber(TelephoneNumberType digits) {
+        StringBuilder phoneNumberBuilder = new StringBuilder();
+
+        phoneNumberBuilder = buildNonNullString(phoneNumberBuilder,
+                digits.getCountryCode(),
+                "",
+                "+s");
+
+        phoneNumberBuilder = buildNonNullString(phoneNumberBuilder,
+                digits.getAreaCode(),
+                " ",
+                "(%s)");
+        phoneNumberBuilder = buildNonNullString(phoneNumberBuilder, digits.getNumber(), " ");
```

Comment:
Does it even make sense to create a phone number string if the number is missing? Should we consider returning a blank string in that case?